### ISSUE TITLE
"Control with Motion Sensor" not appearing

### ISSUE THREAD

**Issue by RadFox34** [WilliamVenner/gmcl_rekinect#3]
So I've got my Xbox One Kinect plugged in and Windows recognizes it. I downloaded the .dll files and placed them all in GarrysMod > garrysmod > lua > bin as instructed. I run rekinector, then open the 64 bit version of Garry's Mod. rekinector say that it successfully injected successfully, but it does not work in Garry's Mod. The option to use the kinect is not present.

**Comment by willcturner** [comment]
You have to join a single or multiplayer, hold c and right click on a regdoll but not all will be supported.

**Comment by RadFox34** [comment]
Yes, that's what I've been doing. I spawn in a default Half-Life 2 ragdoll and the option doesn't appear.

**Comment by JarateIsCools** [comment]
Same issue.

**Comment by Foxache** [comment]
Option to use kinect is present before injecting 
Restarting with injection thru rekinect or CE causes it to stop being availiable on all branches

**Comment by cromwell-scopely** [comment]
If you are having issues with getting Kinect to work and you've followed all of these instructions and encountered similar issues to no avail, the answer is simple:

**Your Kinect Runtime 1.8 did not install correctly. <----- REINSTALL KINECT RUNTIME 1.8 AND INSTALL KINECT DEVELOPER TOOLKIT**

Try downloading the Kinect Developer Toolkit. It will NOT install if you don't have your Kinect Runtime 1.8 correctly installed, and as such, Rekinect will not work. 

Also, some people may have to downgrade to the "DEFAULT PUBLIC VERSION" of Garry's Mod in LIBRARY>PROPERTIRES>GAME VERSIONS & BETAS.

Rekinect works as of August 2026.

**Comment by JarateIsCools** [comment]
I still cannot get it to work I've reinstalled Kinect Runtime 1.8 and Kinect SDK reinstalled Kinect Developer Toolkit and it installs fine.

Nothing shows up in the C menu when I try it on a hl2 ragdoll, running the x86-64 branch by the way.

**Comment by Colt0311** [comment]
Are you using GModpatchtool?
